toot/tui/app.py

```diff
@@ -331,6 +333,28 @@ def _done(instance):
 
         return self.run_in_thread(_load_instance, done_callback=_done)
 
+    def async_load_followed_tags(self):
+        def _load_tag_list():
+            logger.info("Loading tags")
```

Comment:
I removed the logging. There's debug logging when accessing the API so it's redundant here.